src/Paprika/Chain/Blockchain.cs

```diff
@@ -391,7 +396,9 @@ public void SetStorage(in Keccak key, in Keccak address, UInt256 value)
             Set(Key.StorageCell(path, address), payload);
         }
 
-        private void Set(in Key key, in ReadOnlySpan<byte> payload)
+        public bool TryGet(in Key key, out ReadOnlySpanOwner<byte> result) => throw new NotImplementedException("Not implemented yet");
```

Comment:
We already have a `Get` method, why do we need a `TryGet`?

Reply:
This is a subject to debate what to do on the missing one. I assume for now that you'd like to know that it was not there 👀